src/metabase/pulse/render.clj

```diff
@@ -398,37 +398,40 @@
 
 (defn render-pulse-card
   "Render a single CARD for a `Pulse`. DATA is the `:data` from QP results (I think)."
-  [card data]
+  [card result]
```

Comment:
It's probably better just to destructure `result`.

e.g. instead of 
```clojure
(defn render-pulse-card
  [card result]
  (when (:error result)
    ...)
  (let [data (:data result)]
     ...))
```

just do

```clojure
(defn render-pulse-card
  [card {:keys [data error]}]
  (when error
    ...)
  ...)
```